src/helpers/daily-data-types/garmin.tsx

```diff
@@ -0,0 +1,229 @@
+import { FontAwesomeSvgIcon } from "react-fontawesome-svg-icon";
+import { garminActiveCaloriesDataProvider, garminActiveMinutesDataProvider, garminAverageHeartRateDataProvider, garminAverageStressLevelDataProvider, garminAwakeMinutesDataProvider, garminDeepSleepMinutesDataProvider, garminDistanceDataProvider, garminFloorsDataProvider, garminHighStressMinutesDataProvider, garminLightSleepMinutesDataProvider, garminLowStressMinutesDataProvider, garminMaxHeartRateDataProvider, garminMaxStressLevelDataProvider, garminMediumStressMinutesDataProvider, garminMinHeartRateDataProvider, garminRemSleepMinutesDataProvider, garminRestingCaloriesDataProvider, garminRestingHeartRateDataProvider, garminSleepScoreDataProvider, garminStepsDataProvider, garminTotalCaloriesDataProvider, garminTotalSleepMinutesDataProvider, garminTotalStressMinutesDataProvider } from "../daily-data-providers";
+import { DailyDataType, DailyDataTypeDefinition } from "../daily-data-types";
+import { faBed, faBoltLightning, faHeartbeat, faPersonRunning, faRoute, faStairs } from "@fortawesome/free-solid-svg-icons";
+import language from "../language";
+import React from "react";
+import { defaultFormatter, heartRateFormatter, minutesFormatter, sleepYAxisConverter } from "./formatters";
+import { simpleAvailabilityCheck } from "./availability-check";
+
+let garminTypeDefinitions: DailyDataTypeDefinition[] = [
+    {
+        type: DailyDataType.GarminSteps,
+        dataProvider: garminStepsDataProvider,
+        availabilityCheck: simpleAvailabilityCheck("Garmin", ["Daily"]),
```

Comment:
Minor:  The second arg to all of these availability checkers can just be the single string rather than an array.